server/views/includes/index-news.pug

```diff
@@ -0,0 +1,9 @@
+.updates-scroll
+    - const arrayOfObjects = [...allNewsArray]
+    each value in arrayOfObjects
+        p.updates-date #{value.date}
+        p.updates-text #{value.text}
+        if (value.link !== '')
```

Comment:
Could you use the logical and operator (&&) here instead? :) (docs here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND)

Reply:
Noted, updated. Thank you.